### ISSUE TITLE
--dry-run and --write-sql does not output the SQL statement to display.

### ISSUE THREAD

**Issue by ichiro-nagata** [doctrine/DoctrineORMModule#290]
![screen shot 2013-12-05 at 1 27 33 am](https://f.cloud.github.com/assets/885328/1679092/d2b09b36-5d45-11e3-9a50-c108642e61b2.png)

Cause is outputWriter of doctrine/migrations is the closure of the null by default.

How can I set the closure of its own?

To try, I tried to modified as follows.

``` PHP
'migrations_configuration' => array(
    'orm_default' => array(
        'outputWriter' => call_user_func(function() {
            $output = new ConsoleOutput();
            return new OutputWriter(function($message) use ($output) {
                $output->writeln($message);
            });

        }),
    ),
),
```

MigrationsConfigurationFactory.php

``` PHP
$configuration = new Configuration($connection, $migrationsConfig['outputWriter']);
```

![screen shot 2013-12-05 at 2 04 47 am](https://f.cloud.github.com/assets/885328/1679093/d6638c3e-5d45-11e3-8d52-185de6baf349.png)

**Comment by snapshotpl** [comment]
--dry-run and --query-time still doesn't work :(

**Comment by gianarb** [comment]
Hi! can you try this module please?
https://github.com/gianarb/DoctrineMigrationsModule

Referance #401 :)

**Comment by Gemorroj** [comment]
:+1: Had to spend a half day before I found this bug =(

**Comment by TomHAnderson** [comment]
Moved to migrations: https://github.com/doctrine/migrations/issues/530

**Comment by TomHAnderson** [comment]
Closed due to lack of activity.

**Comment by svycka** [comment]
@TomHAnderson but it's still a bug even no one volunteered to fix it

**Comment by TomHAnderson** [comment]
Fixed by #528
